examples/up.conf.example

```diff
@@ -65,6 +65,82 @@
   interval   = "2m"
   timeout    = "10s"
 
+[datadog]
+  # How often to poll UniFi and report to Datadog.
+  interval: "2m"
+
+  # To disable this output plugin
+  disable: false
+
+  # Datadog Custom Options
+
+  # address to talk to the datadog agent, by default this uses the local statsd UDP interface
+  # address: "..."
+
+  # namespace to prepend to all data
+  # namespace: ""
+
+  # tags to append to all data
+  # tags:
+  #  - foo
+
+  # max_bytes_per_payload is the maximum number of bytes a single payload will contain.
+  # The magic value 0 will set the option to the optimal size for the transport
+  # protocol used when creating the client: 1432 for UDP and 8192 for UDS.
+  # max_bytes_per_payload: 0
+
+  # max_messages_per_payload is the maximum number of metrics, events and/or service checks a single payload will contain.
+  # This option can be set to `1` to create an unbuffered client.
+  # max_messages_per_payload: 0
+
+  # BufferPoolSize is the size of the pool of buffers in number of buffers.
+  # The magic value 0 will set the option to the optimal size for the transport
+  # protocol used when creating the client: 2048 for UDP and 512 for UDS.
+  # buffer_pool_size: 0
+
+  # buffer_flush_interval is the interval after which the current buffer will get flushed.
+  # buffer_flush_interval: 0
+
+  # buffer_shard_count is the number of buffer "shards" that will be used.
+  # Those shards allows the use of multiple buffers at the same time to reduce
+  # lock contention.
+  # buffer_shard_count: 0
```

Comment:
The more I read into these settings the more I scratch my head. How much tuning is needed here? Are larger sites with more data points going to need to tweak these things? Maybe it's my lack of datadog experience, but I'm a little blown away at the number of tunables here.

Reply:
yeah if there was a *large* number of sites from a single collector this could be problematic - alternatively we could drop these examples and leave them in the datadogunifi readme for advanced users.